inc/storefront-template-functions.php

```diff
@@ -183,8 +183,8 @@ function storefront_site_branding() {
 	 */
 	function storefront_primary_navigation() {
 		?>
-		<nav id="site-navigation" class="main-navigation" role="navigation" aria-label="<?php esc_html_e( 'Primary Navigation', 'storefront' ); ?>">
-		<button class="menu-toggle" aria-controls="primary-navigation" aria-expanded="false"><span><?php echo esc_attr( apply_filters( 'storefront_menu_toggle_text', __( 'Menu', 'storefront' ) ) ); ?></span></button>
+		<div id="site-navigation" class="main-navigation" role="navigation" aria-label="<?php esc_html_e( 'Primary Navigation', 'storefront' ); ?>">
```

Comment:
> Was it harmfull to use <div> in this context before HTML5

No because `nav` didn't exist / wasn't recognised then xD

I think we should leave it as `nav`, I also think we should include the header because;

1. It's good for screen readers considering we have multiple navs
2. Valid html feels good.